avm/res/automation/automation-account/credential/main.bicep

```diff
@@ -0,0 +1,42 @@
+metadata name = 'Automation Account Credentials'
+metadata description = 'This module deploys Azure Automation Account Credentials.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. Name of the Automation Account credential.')
+param credentialName string
```

Comment:
It's usually a convention that there's always just a `name` parameter as it should be clear that a `name` parameter for a `credential` module implies that you're setting the credential's name :) 

```suggestion
param name string
```

Reply:
Please note, this must be updated in several other locations too and requires a regeneration of the files

Reply:
parameter name updated

Reply:
Hey @elanzel, I suspect you may have forgotten to push your commit 😉

Reply:
updated